package.json

```diff
@@ -135,7 +135,7 @@
     "babel-preset-stage-0": "6.24.1",
     "cross-env": "5.1.1",
     "css-loader": "0.28.7",
-    "electron": "1.8.4",
+    "electron": "1.7.11",
```

Comment:
Whats this about?

Reply:
I ran into a problem when cloning the application and running NPM install. NPM installed electron 1.8.4 when the readme stated that 1.7.11 should be used. Once I updated the electron version the application ran without issues. Are you running 1.8.4 without problems?

Reply:
I haven't run into any problems thus far, but @FredrikOseberg is correct; the README needs to be updated.

Reply:
<img width="1050" alt="screen shot 2018-07-08 at 13 53 03" src="https://user-images.githubusercontent.com/16081982/42419516-b691d974-82b6-11e8-8ebc-0f2654de539c.png">

Reply:
Please revert this change 👆